module_1/src/com/epam/cycles/CyclesTaskD.java

```diff
@@ -0,0 +1,17 @@
+package com.epam.cycles;
+import java.math.BigInteger;
+
+public class CyclesTaskD {
+
+    private static final int AMOUNT_OF_NUMBERS = 200;
+
+    public static void main(String[] args) {
+        BigInteger product = new BigInteger("1");
+        BigInteger squareNumber;
+        for (int i = 1; i <= AMOUNT_OF_NUMBERS; i++) {
```

Comment:
I think it is more logical to start from number - 2 because you already assigned your product to 1
```suggestion
        for (int i = 2; i <= AMOUNT_OF_NUMBERS; i++) {
```